nutkit/backend/backend.py

```diff
@@ -136,6 +136,24 @@ def send_and_receive(self, req, timeout=None, hooks=None):
         self.send(req, hooks=hooks)
         return self.receive(timeout, hooks=hooks)
 
+    def _check_system_support(self, type_, meta):
```

Comment:
do you think it's worth introducing a general system check API now? what kind of checks do you foresee?

Reply:
Maybe as JS if it knows the difference between `0.0` and `0` 😐. I was just thinking that I don't want to introduce a message just for checking if a certain timezone is supported. That felt way to specific.

Reply:
I usually try to generalize something once I've got evidence of several specializations but YMMV

you could argue we can start with the simple, very specific timezone check and generalize it afterwards
that being said, such a generalization, if done at a later stage, would likely be a breaking change that would affect all testkit clients

Reply:
Yes. If it was just TestKit that's effected I wouldn't have done this. But as you say it affects the protocol and hence all backends. But I don't have a strong opinion on that. If you have, I can turn this into a specific message pair.